File: lanaya-business-service/src/main/resources/static/js/service/businessBrandService.js
app.service('businessBrandService', function ($http) {
    this.pageByClause = function (entity) {
        return $http.post('../businessBrand/pageByClause', entity);
    };

    this.insertSelective = function (entity) {
        return $http.post('../businessBrand/insertSelective', entity);
    };

    this.updateByPrimaryKeySelective = function () {

        return $http.post('../businessBrand/updateByPrimaryKeySelective', $scope.entity);
    };

    this.selectByPrimaryKey = function (id) {
        return $http.post('../businessBrand/selectByPrimaryKey', json({id:id}));
    };
    this.deleteByPrimaryKeyBatch = function () {
        return $http.post('../businessBrand/deleteByPrimaryKeyBatch', '');
    };
});

File: lanaya-business/src/main/java/cn/lanaya/business/service/BusinessBrandService.java
package cn.lanaya.business.service;

import cn.lanaya.business.entity.BusinessBrand;
import cn.lanaya.business.vo.BusinessBrandVO;
import cn.lanaya.common.bean.PageQO;
import com.github.pagehelper.PageInfo;
import java.util.List;

public interface BusinessBrandService {
    int countByClause(BusinessBrandVO record);

    int deleteByClause(BusinessBrandVO record);

    int deleteByPrimaryKey(Long id);

    int insert(BusinessBrand record);

    int insertSelective(BusinessBrand record);

    List<BusinessBrand> selectByClause(BusinessBrand record);

    BusinessBrand selectByPrimaryKey(Long id);

    int updateByClauseSelective(BusinessBrand entity, BusinessBrandVO param);

    int updateByPrimaryKeySelective(BusinessBrand record);

    int updateByPrimaryKey(BusinessBrand record);

    PageInfo<BusinessBrandVO> pageByClause(PageQO page);

    int deleteByPrimaryKeyBatch(List<BusinessBrandVO> list);

    int insertSelectiveBatch(List<BusinessBrand> list);
}